SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec

```diff
@@ -0,0 +1,50 @@
+Summary:        Package to deploy azurelinux-sysinfo service
+Name:           azurelinux-sysinfo
+Version:        %{azl}.0
+Release:        1%{?dist}
+License:        GPLv2+
```

Comment:
default MSFT license should be MIT unless there's a reason to be something else. I don't think that's the case here, but please do correct me if I'm wrong.